src/synapse/api.py

```diff
@@ -193,6 +217,11 @@ def get_version() -> str:
         res = session.get(VERSION_URL, timeout=10)
         res.raise_for_status()
         res_json = res.json()
+        if not isinstance(res_json, dict):
```

Comment:
See example above for how to simplify this

Reply:
Changed :)